### DISCUSSION TITLE
Can I fix the CORS from side the beckand

### DISCUSSION THREAD

**Discussion by happyDuke96** [chatwoot/chatwoot#6278] [Chatwoot APIs] [answered]
Hello Chatwoot Team,
when using selfhost, this url does not work  http://myhostname:3000/api/v1/accounts/12/conversations,
it gives a cors error,
Access to XMLHttpRequest at '**http://myhostname:3000/api/v1/accounts/12/conversations**' from origin 
'**http://localhost:3000**' has been blocked by CORS policy: Response to preflight request doesn't pass access control check: 
No 'Access-Control-Allow-Origin' header is present on the requested resource.
**I know this is a known bug
the browser blocks requests from different domains, and it can be bypassed through a proxy request, but is there another solution from side the backend**. I correctly configured the frontend url also tried with your template through this one, how can I fix it, can I add some kind of environment or something like in the rails console
![Inkedcors_error](https://user-images.githubusercontent.com/105979703/213157464-9f12787a-b60a-4adf-8ca9-8850531e4ddc.jpg)

**Comment by tejaswinichile** [comment, marked as answer]
You can change your CORS policy according to https://github.com/cyu/rack-cors and test the application.

**Reply by happyDuke96** [reply]
Thank you
